src/github/views.ts

```diff
@@ -96,6 +96,7 @@ export interface PullRequest extends Issue {
 	lastReviewType?: ReviewType;
 	revertable?: boolean;
 	busy?: boolean;
+	closingIssues: Pick<Issue, 'title' | 'number' | 'state'>[];

```

Comment:
`Pick<Issue, 'title' | 'number' | 'state'>` is used in more than one place, can you refactor it into an `IssueReference` type in this file?